lib/tokenize.cpp

```diff
@@ -6749,7 +6749,7 @@ void Tokenizer::simplifyFunctionParameters()
             // First step: Get list of argument names in parentheses
             std::map<std::string, Token *> argumentNames;
             bool bailOut = false;
-            Token * tokparam = nullptr;
+            const Token * tokparam = nullptr;
```

Comment:
It seems strange we can detect this now. Looks like some underlying false negative.